### PR TITLE
Update download-software-licenses-csp.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about GA

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23711

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/download-software-licenses-csp.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ This article explains how to download software and product license keys for perp
 
 You must be a Global admin to do the steps in this article. For more information, see [About admin roles](../add-users/about-admin-roles.md).
 
+> [!NOTE]
+> If you are a partner, you need to log in to the customer tenant with an admin account created in this tenant.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/download-software-licenses-csp.md]:
```suggestion
> If you are a partner, you need to sign in to the customer's organization with an admin account created in the organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by oradcliffe** [comment]
Bump - has not been updated.  Can someone please confirm this is correct, that the Global Admin account must be homed in the customer tenant (ie a CSP partner with Admin on Behalf Of into customer tenant cannot perform this task), and then update the doc accordingly?

**Comment by oradcliffe** [comment]
Page is still not updated

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder. Thanks

**Comment by oradcliffe** [comment]
Looking for an update.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT This is long pending, please review, comment or merge. Thanks

**Comment by msbemba** [comment]
Fixes https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23702 as well

**Review by cmcatee-MSFT — COMMENTED** [review]
comment removed

**Pull request merged**